src/services/applicationGateway/schema.graphql

```diff
@@ -0,0 +1,1273 @@
+  destinationAddressPrefixes: [String] @search(by: [hash, regexp])
+  destinationApplicationSecurityGroups: [azureApplicationGatewayApplicationSecurityGroup]
+  sourcePortRanges: [String] @search(by: [hash, regexp])
+  destinationPortRanges: [String] @search(by: [hash, regexp])
+  access: String @search(by: [hash, regexp])
+  priority: Int @search
+  direction: String @search(by: [hash, regexp])
+  provisioningState: String @search(by: [hash, regexp])
+}
+
+type azureApplicationGatewayNetworkInterfaceDnsSettings
+  @generate(
+    query: { get: false, query: true, aggregate: false }
+    mutation: { add: false, delete: false }
+    subscription: false
+  ) {
+  dnsServers: [String] @search(by: [hash, regexp])
+  appliedDnsServers: [String] @search(by: [hash, regexp])
+  internalDnsNameLabel: String @search(by: [hash, regexp])
+  internalFqdn: String @search(by: [hash, regexp])
+  internalDomainNameSuffix: String @search(by: [hash, regexp])
+}
+
+type azureApplicationGatewayPrivateLinkServiceIpConfiguration
+  @generate(
+    query: { get: false, query: true, aggregate: false }
+    mutation: { add: false, delete: false }
+    subscription: false
+  ) {
+  id: String @search(by: [hash, regexp])
+  name: String @search(by: [hash, regexp])
+  type: String @search(by: [hash, regexp])
+  privateIPAddress: String @search(by: [hash, regexp])
+  privateIPAllocationMethod: String @search(by: [hash, regexp])
+  subnet: azureApplicationGatewaySubnet
+  primary: Boolean @search
+  provisioningState: String @search(by: [hash, regexp])
+  privateIPAddressVersion: String @search(by: [hash, regexp])
+}
+
+type azureApplicationGatewayApplicationGatewayPrivateEndpointConnection
+  @generate(
+    query: { get: false, query: true, aggregate: false }
+    mutation: { add: false, delete: false }
+    subscription: false
+  ) {
+      id: String @search(by: [hash, regexp])
+  name: String @search(by: [hash, regexp])
+  type: String @search(by: [hash, regexp])
+  privateEndpoint: azureApplicationGatewayPrivateEndpoint
+  privateLinkServiceConnectionState: azureApplicationGatewayPrivateLinkServiceConnectionState
+  provisioningState: String @search(by: [hash, regexp])
+  linkIdentifier: String @search(by: [hash, regexp])
+}
+
+type azureApplicationGatewayPrivateLinkService
+  @generate(
+    query: { get: false, query: true, aggregate: false }
+    mutation: { add: false, delete: false }
+    subscription: false
+  ) {
+    id: String @search(by: [hash, regexp])
+    extendedLocation: azureApplicationGatewayExtendedLocation
+    loadBalancerFrontendIpConfigurations: [azureApplicationGatewayFrontendIPConfiguration]
+    ipConfigurations: [azureApplicationGatewayPrivateLinkServiceIpConfiguration]
+    networkInterfaces: [azureApplicationGatewayNetworkInterface]
+    provisioningState: String @search(by: [hash, regexp])
+    privateEndpointConnections: [azureApplicationGatewayApplicationGatewayPrivateEndpointConnection]
```

Comment:
@m-pizarro can you take a look at this? I imagine some of these names are too long to support the sql def

Reply:
sure, I'll take a look

Reply:
@tyler-dunkel I've been identifying the tables that need to be renamed but from what I see a lot have been added and most should be renamed. How about we go ahead with this merge and then I'll update my branch to fix all of them?

Reply:
sounds good to me!